contributing/building-an-amp-extension.md

```diff
@@ -253,9 +253,11 @@ Element styles are loaded when the element script itself is included in
 an AMP doc. You tell AMP which CSS belongs to this element when
 registering the element (see below).
 
-Class names prefixed with `-amp-` are considered private and
+Class names prefixed with `-amp-` and `i-amphtml` are considered private and
 publishers are not allowed to use to customize (enforced by AMP
-validator).
+validator). Components should prefix any public css classes with `amp-` and
+documented in the component-specific `.md` file. All css classes in component
```

Comment:
This is a little unclear. Make it a separate paragraph and decouple the idea of  public vs. private.

nit: `CSS` in uppercase.

/cc @bpaduch